### PR TITLE
Supported storage classes for VMware>CNV import

### PR DESCRIPTION
**Pull request by apinnick**
https://bugzilla.redhat.com/show_bug.cgi?id=1859302

CP for 4.5 and 4.6

### CODE REVIEW

File: modules/virt-importing-vm-wizard.adoc [hunk 1]
```diff
@@ -101,7 +102,20 @@ ifdef::virt-importing-vmware-vm[]
...
+
+*** VM disk: *ocs-storagecluster-ceph-rbd (ceph-rbd)*
+*** v2v-conversion-template disk: *ocs-storagecluster-ceph-rbd (ceph-rbd)*
++
+_or_
+
+*** VM disk: *NFS*
+*** v2v-conversion-template disk: *hostpath-provisioner (HPP)*
++
+Other storage classes may work although they are not officially supported.
```
Comment (ousleyp) [inline modules/virt-importing-vm-wizard.adoc:117]: I think this sentence might encourage customers to try unsupported storage classes, so I would personally remove it. 

If you keep it in, s/may/might (ISG)
s/work although/work, but

File: modules/virt-updating-imported-vm-network-name.adoc [hunk 2]
```diff
@@ -3,20 +3,20 @@
 // * virt/virtual_machines/importing_vms/virt-importing-vmware-vm.adoc
 
 [id="virt-updating-imported-vmware-vm-network-name_{context}"]
-= Updating the imported VMware virtual machine's NIC name
+= Updating the NIC name
```
Comment (ousleyp) [inline modules/virt-updating-imported-vm-network-name.adoc:6]: The title and the file name should align more closely. Maybe "Updating the NIC name of an imported virtual machine"?

File: modules/virt-updating-imported-vm-network-name.adoc [hunk 3]
```diff
@@ -3,20 +3,20 @@
...
 
 You must update the NIC name of a virtual machine imported from VMware to conform to {VirtProductName} naming conventions.
```
Comment (ousleyp) [inline modules/virt-updating-imported-vm-network-name.adoc:8]: s/imported/that you imported

File: modules/virt-updating-imported-vm-network-name.adoc [hunk 4]
```diff
@@ -3,20 +3,20 @@
...
-. Go to the `/etc/sysconfig/network-scripts` directory.
-. Change the network configuration file name to `ifcfg-eth0`:
+. Navigate to the `/etc/sysconfig/network-scripts` directory.
+. Rename the network configuration file:
 +
 ----
 $ mv vmnic0 ifcfg-eth0 <1>
 ----
-<1> Additional network configuration files are numbered sequentially, for example, `ifcfg-eth1`, `ifcfg-eth2`.
+<1> The first network configuration file is called `ifcfg-eth0`. Additional network configuration files are numbered sequentially, for example, `ifcfg-eth1`, `ifcfg-eth2`.
```
Comment (ousleyp) [inline modules/virt-updating-imported-vm-network-name.adoc:19]: s/called/named
s/sequentially, for example,/sequentially. For example:

**Review by ousleyp — COMMENTED** [review]
This is looking good but I have a few clarity nits and suggestions. Thanks, Avital!

File: modules/virt-updating-imported-vm-network-name.adoc (hunk 4 above)
Comment (apinnick) [reply modules/virt-updating-imported-vm-network-name.adoc:19]: First, yes. Second, no. I don't like sentence fragments. It's better this way.

File: modules/virt-importing-vm-wizard.adoc (hunk 1 above)
Comment (apinnick) [reply modules/virt-importing-vm-wizard.adoc:117]: We don't really care if they try it. If it works, that's great. If it doesn't, the import won't work and they'll have to use a different storage class. They won't break anything by using an unsupported storage class.

**Comment by apinnick** [comment]
@ousleyp Can this be merged?

File: modules/virt-importing-vm-wizard.adoc (hunk 1 above)
Comment (ousleyp) [reply modules/virt-importing-vm-wizard.adoc:117]: @djfjeff Hey, can you please confirm that this is okay with you and the support team? Thanks!

**Comment by ousleyp** [comment]
> @ousleyp Can this be merged?

I just want to make sure that the support team approves the unsupported storage class line before merging.

File: modules/virt-importing-vm-wizard.adoc (hunk 1 above)
Comment (djfjeff) [reply modules/virt-importing-vm-wizard.adoc:117]: @ousleyp I am OK with that (including the suggestions you made 2 comments above). I think we should have this mention and not leave this unclear.

Comment (ousleyp) [reply modules/virt-importing-vm-wizard.adoc:117]: Thanks @djfjeff! 

@apinnick, I'll be happy to merge this PR after you change this line to "Other storage classes might work, but they are not officially supported."

Comment (apinnick) [reply modules/virt-importing-vm-wizard.adoc]: @ousleyp Done

**Pull request merged**